app/controllers/registrations_controller.rb

```diff
@@ -46,6 +46,7 @@ def update_first_user_permissions(resource)
     resource.skip_confirmation!
     Settings::General.waiting_on_first_user = false
     Users::CreateMascotAccount.call
+    Discover::RegisterWorker.perform_async # Register Forem instance on https://discover.forem.com
```

Comment:
The first user might register before the Forem _is set up on the final domain_. So I think here, there's a good chance this will register as `forem-3232dsdsd.forem.cloud` or whatever, unless I'm mistaken.

It seems like the place for this to go is on the final steps _of the set up_ phase _and/or_ when the proper `Settings::General.app_domain` is set. cc: @Ridhwana on all this.

There could be additional nuance to where this is called, and I think it could go along with a check-box allowing the admin to opt out in the moment if they don't want to be discovered.

I'm thinking maybe ship all of the functional code, but remove this call and the adjacent test for Admin Experience to appropriately integrate into their work?

Reply:
I agree with this rationale - I've added it to our board and will ask @anujbhavsar96 to help figure out the best place for it.